setup.py

```diff
@@ -50,6 +54,7 @@ def read(filename):
         irclog2html = irclog2html.irclog2html:main
         logs2html = irclog2html.logs2html:main
         irclogsearch = irclog2html.irclogsearch:main
+        serve = irclog2html.irclogsearch:serve
```

Comment:
This is a very generic name.  If I pip install irclog2html, I'd want something more specific on my $PATH than just "serve".

How about "irclogserver"?